plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/password-strength-meter/index.tsx

```diff
@@ -29,13 +29,51 @@ const scoreDescriptions = [
 	__( 'Very strong', 'woocommerce' ),
 ];
 
+export const getPasswordStrength = ( password: string ) => {
+	if ( typeof window.zxcvbn === 'undefined' ) {
```

Comment:
Are we making zxcvbn optional?

Reply:
I assume it's not loaded in tests.

Reply:
Why not install it via npm and add it to our dependency extraction plugin?

Reply:
It is optional; if its there, great, we use it. If its missing or dequeued we have a good fallback in place.

Reply:
`zxcvbn-async` is a dependency of the checkout block. It's async because its heavy.